mediafile.py

```diff
@@ -2326,6 +2327,20 @@ def bitrate(self):
                 return 0
             return int(self.filesize * 8 / self.length)
 
+    @property
+    def bitrate_mode(self):
+        """The mode of the bitrate used in the audio coding (a string).
```

Comment:
Perhaps we should mention what the current possibilities are? (Without promising that they are the only possibilities this field will ever contain.)

Reply:
Sure, I have updated the docstring